### ISSUE TITLE
Blackmagic GPI Interface

### ISSUE THREAD

**Issue by endlichmitnamen** [bitfocus/companion-module-requests#1040]
- [ ] **I have researched the list of existing Companion modules and requests and have determined this has not yet been requested**

The name of the device, hardware, or software you would like to control:
Blackmagic Design Tally and GPI Interface

What you would like to be able to make it do from Companion:
Input and Output Interface for GPIs and GPOs from/to other devices. Would be great if the interface inputs (short on GPI) could trigger the stream deck buttons, so it would be possible to trigger functions not only internal from streamdeck but also from a pushbutton outside. for example switching signals on a videomatrix can then be done from streamdeck or in parallel from the joystick override of camera-control-panels (the joystick override function is a standard on every professional camera controlpanel, in fact, hitting the joystick is sending a contact-closure to an external GPIO-port)

Direct links or attachments to the ethernet control protocol or API:
can't find any informations from vendor directly, but the protocol should be in very close relation to the feedback function for atem switchers (for example "id":"gnQunl_l31","instance_id":"HaFuPqMAb") and/or videohub ({"type":"selected_source","options":{"bg":16777215,"fg":0,"input":2},"id":"J-mTG07Pvk","instance_id":"uENtisV4-"})

**Comment by Julusian** [comment]
Duplicate of bitfocus/companion-module-bmd-gpi-and-tally-interface#1
